lib/core/boot/boot.ts

```diff
@@ -79,11 +84,38 @@ export const bootstrap = (): FluxifyServer => {
 						if (targetRoute.schema.body) body = targetRoute.schema.body.parse(body);
 					}
 
+					if (useCache) {
+						if (global.server.cache.length > config.cacheLimit) {
+							global.server.cache.pop();
+						}
+						global.server.cache = global.server.cache.filter((entry) => entry.exp > Date.now());
+						const hit = global.server.cache.find(
+							(entry) => entry.url === request.url && entry.jwt === (jwt as { id?: string })?.id,
```

Comment:
Why is only the id getting checked, cached?

Reply:
I sure hope that the user id is unique enough. also performance reasons.